src/clone.c

```diff
@@ -382,11 +382,12 @@ int git_clone__should_clone_local(const char *url_or_path, git_clone_local_t loc
 	return is_local;
 }
 
-int git_clone(
+static int git__clone(
```

Comment:
`clone` would be sufficient, I guess :)

Reply:
I think you got me there, I just had the Linux buildbots remind me that it's some syscall 🤣.